data/hooks/conf_regen/10-apt

```diff
@@ -17,19 +17,16 @@ Pin-Priority: -1" >> "${pending_dir}/etc/apt/preferences.d/extra_php_version"
     done
 
         echo "
-# Yes !
-# This is what's preventing you from installing apache2 !
-#
-# Maybe take two fucking minutes to realize that if you try to install
-# apache2, this will break nginx and break the entire YunoHost ecosystem.
-# on your server.
-#
-# So, *NO*
-# DO NOT do this.
-# DO NOT remove these lines.
-#
-# I warned you. I WARNED YOU! But did you listen to me?
-# Oooooh, noooo. You knew it all, didn't you?
+
+# PLEASE READ THIS WARNING AND DON'T EDIT THIS FILE
+
+# You are probably reading this file cause you tryed to install apache2 or 
```

Comment:
`because you tried` c’est mieux je pense que `cause you tryed`

Reply:
done